addons/main/script_macros.hpp

```diff
@@ -9,19 +9,18 @@
 #define DFUNC(var1) TRIPLES(ADDON,fnc,var1)
 #define DEFUNC(var1,var2) TRIPLES(DOUBLES(PREFIX,var1),fnc,var2)
 
-#define QFUNC(var1) QUOTE(DFUNC(var1))
-#define QEFUNC(var1,var2) QUOTE(DEFUNC(var1,var2))

```

Comment:
Good catch, when using `#define ENABLE_CALLSTACK` it throws an error when using QFUNC.
So I'm fairly sure we do want to undef and redefine QFUNC using DFUNC.